templates/ehub-existing-setup.json

```diff
@@ -0,0 +1,267 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "Name": {
```

Comment:
May worth renaming this as "Function Name" to make sure people don't think this is the ehub name?

Reply:
Let's rename it to `Application Name`